tests/test_s3bucket.py

```diff
@@ -0,0 +1,86 @@
+
+import boto3
+import pytest
+from botocore.exceptions import ClientError
+from moto import mock_s3
+
+
+from prefect_aws import AwsCredentials, MinIOCredentials, S3Bucket
+
+aws_creds_block = AwsCredentials(
+    aws_access_key_id="testing", aws_secret_access_key="testing"
+)
+
+minio_creds_block = MinIOCredentials(
+    minio_root_user="testing", minio_root_password="testing"
+)
```

Comment:
I think it's more standard to use @pytest.fixture for these
https://docs.pytest.org/en/6.2.x/fixture.html#what-fixtures-are

```suggestion
@pytest.fixture
def minio_creds_block():
    return MinIOCredentials(
        minio_root_user="testing", minio_root_password="testing"
    )
```

Reply:
good idea